packages/lesswrong/server/collections/users/serverHelpers.ts

```diff
@@ -0,0 +1,80 @@
+import { MAX_ALLOWED_CONTACTS_BEFORE_FLAG } from "@/lib/collections/moderatorActions/constants";
+import AutomatedContentEvaluations from "../automatedContentEvaluations/collection";
+import { DatabasePublicSetting } from "@/lib/publicSettings";
+import { getCurrentContentCount } from "@/lib/collections/moderatorActions/helpers";
+
+type ReasonNoReviewNeeded = "alreadyApproved"|"noReview"|"llmRejected"
+type ReasonReviewIsNeeded = "mapLocation"|"firstPost"|"firstComment"|"contactedTooManyUsers"|"bio"|"profileImage"|"newContent";
+type GetReasonForReviewResult =
+    { needsReview: false, reason: ReasonNoReviewNeeded }
+  | { needsReview: true, reason: ReasonReviewIsNeeded }
+type ReasonForInitialReview = Exclude<ReasonReviewIsNeeded, 'newContent'> 
+const reviewReasonsSetting = 
+  new DatabasePublicSetting<Array<ReasonForInitialReview>>(
+    'moderation.reasonsForInitialReview', 
+    ['firstPost', 'firstComment', 'contactedTooManyUsers', 'bio', 'profileImage']
+  )
+
+
+const checkIfLLMRejected = async (newDocument: DbPost|DbComment): Promise<boolean> => {
+  if (!newDocument.contents_latest) return false;
+  const isLLMRejected = await AutomatedContentEvaluations.findOne({
+    revisionId: newDocument.contents_latest,
+    score: {$gte: 0.5},
+    aiChoice: "review",
+  });
+  return !!isLLMRejected;
+}
+
+export async function getReasonForReview(user: DbUser|SunshineUsersList, newDocument?: DbPost|DbComment): Promise<GetReasonForReviewResult>
+{
+  /**
```

Comment:
`/**` (with an extra star) is a marker for special comments that are automatically extracted from documentation, which must be directly above a function or field declaration. This is a regular comment so it should use `/*` (one star).